app/controllers/casework/documents_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Casework
+  class DocumentsController < Casework::BaseController
+    before_action :set_crime_application, :set_document
+
+    class CannotDownloadUnlessAssigned < StandardError; end
+    class CannotDownloadDocNotPartOfApp < StandardError; end
```

Comment:
Let me know if this name makes sense? I struggled to keep the name within the rubocop character limit but also convey enough meaning..

Reply:
I think this is good - it is clear what it is and squeezes a lot of info in to a relitively few characters.